README.md

```diff
@@ -16,6 +16,9 @@
 	<a href="https://codecov.io/gh/JoshuaKGoldberg/ts-api-utils" target="_blank">
 		<img alt="Codecov Test Coverage" src="https://codecov.io/gh/JoshuaKGoldberg/ts-api-utils/branch/main/graph/badge.svg?token=eVIFY4MhfQ"/>
 	</a>
+	<a href="#" target="_blank">
+		<img alt="Documentation Coverage" src="./docs/coverage.svg"/>
```

Comment:
[Request] 🤔 I'm not sure how to get this to work. `docs/` is gitignored so this is going to look odd on github.com. Is the right thing linking to the deployed GitHub site?

Reply:
Ah right. An oversight by me. I'll fix that up.